### ISSUE TITLE
Ponder EG sustain behavior on modulation

### ISSUE THREAD

**Issue by Andreya-Autumn** [surge-synthesizer/surge#6874]
As of 1.2, EG sustain params behave in two ways: 

LFOEG's can not have sustain modulated during the sustain phase. Modulation will be in effect during decay, as the decay approaches the moving target, but once sustain phase begins the value is held as discussed here: https://github.com/surge-synthesizer/surge-rack/issues/858

FEG and AEG can have sustain modulated, also once the sustain phase begins. The modulation is slew-limited by the decay rate parameter though. EvilDragon points out that this is probably congruent with many analog designs (in which decay mat quite literally be a slew limiter on sustain value), but I don't know if this is expected behavior in a digital synth like Surge. 

Changes here would be breaking obviously. So for XT2:

A: Decide on what behavior we would really want.
B: Make sure that behavior is consistent across AEG/FEG/LFOEG

**Comment by mkruselj** [comment]
Behavior of modulating sustain as is in FEG/AEG is expected and should happen across all envelopes.

**Comment by baconpaul** [comment]
This is not hard to fix if we make a breaking change.
This is quite hard to fix if we want to make a backward compatible change since the envelope won't know where it was found.
Do we want to make a breaking change? 

Obviously the lfo eg is wrong though. If you just tagged this to say that and we shouldn't do anything I agree! :)

**Comment by mkruselj** [comment]
Nah, it's marked XT 2.0 and a breaking change. We can just do it properly when it's time. I just wanted to leave a comment to the A/B options Andreya posed in the opening post.
